internal/node/node_repositories.bzl

```diff
@@ -141,6 +141,12 @@ Note that the dependency installation scripts will run in each subpackage indica
 
 # TODO(kgreenek): Add arm64 versions for all of these.
 _ATTRS = {
+    "node_download_auth": attr.string_dict(
+        default = {},
+        doc = """auth to use for all url requests
+fx: {\"type\": \"basic\", \"login\": \"<UserName>\", \"password\": \"<Password>\" }
```

Comment:
what does `fx` mean here?

Reply:
That is bad English I'm afraid, it is meant as for example. Which is very close to the Danish way of writing it.
I guess the English way of writing it is e.g. I will change this to `Example:`